src/__tests__/pluginTester.js

```diff
@@ -0,0 +1,36 @@
+import { JestHook } from 'jest-watcher';
```

Comment:
what do you think about exporting this from `jest-watcher`. Eslint has great success by exporting helpers making it a breeze to test - watch plugins (and runners) might benefit from the same approach in the long run?

Reply:
I agree!!

I feel that it is a bit too early to move it... I'll work on making this plugin configurable(`key`, `prompt`, `search-strategy`, etc) and that should help me understand if this was the right abstraction for the tester.

Reply:
Yeah, sounds like a great idea